app/models/asset_field_type.rb

```diff
@@ -119,6 +125,7 @@ def validate_asset_content_type
   end
 
   def style_urls
+    return unless image?
```

Comment:
Yeah and it matches `def extract_dimensions` which also uses the `image?` method so I was trying to match style.